src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/CaseWorkerRefController.java

```diff
@@ -145,15 +146,24 @@
             throw new InvalidRequestException("Caseworker Profiles Request is empty");
         }
 
+        CaseWorkerProfileCreationResponse.CaseWorkerProfileCreationResponseBuilder caseWorkerProfileCreationResponse =
+                CaseWorkerProfileCreationResponse
+                        .builder();
         List<CaseWorkerProfile> processedCwProfiles =
                 caseWorkerService.processCaseWorkerProfiles(caseWorkersProfileCreationRequest);
 
         if (!processedCwProfiles.isEmpty()) {
             caseWorkerService.publishCaseWorkerDataToTopic(processedCwProfiles);
```

Comment:
CollectionUtils.isNotEmpty. As ! might come sonar Minor

Reply:
Did not find anything reported by sonar as such - https://sonarcloud.io/project/issues?id=uk.gov.hmcts.reform%3Areform%3A%3Ard-caseworker-ref-api&pullRequest=124&resolved=false :)